src/ZkFold/Symbolic/Compiler/ArithmeticCircuit/Instance.hs

```diff
@@ -133,9 +135,16 @@ instance {-# OVERLAPPING #-} SymbolicData a x => Conditional (Bool (ArithmeticCi
 
 -- TODO: make a proper implementation of Arbitrary
 instance Arithmetic a => Arbitrary (ArithmeticCircuit a) where
-    arbitrary = do
-        let ac = mempty { acOutput = 1} * mempty { acOutput = 2}
-        return ac
+    arbitrary = frequency [
+        (1, return mempty { acOutput = 1}),
```

Comment:
This only creates a circuit with one input. Let's generalize it to at least up to `100` inputs.